src/components/SearchModal/CurrencyList/CurrencyList.tsx

```diff
@@ -232,11 +234,21 @@ export default function CurrencyList({
 
       const showImport = index > currencies.length
 
-      if (showImport && token) {
+      if (isLoading) {
+        return (
+          <LoadingRows>
+            <div />
```

Comment:
Why are there divs here?

Reply:
The divs are the actual loading bars (each div is 1 bar). See https://github.com/Uniswap/interface/blob/main/src/pages/Pool/index.tsx#L110 for similar usage